pkg/kubelet/sysctl/whitelist.go

```diff
@@ -26,23 +26,23 @@ import (
 )
 
 const (
-	AnnotationInvalidReason = "InvalidSysctlAnnotation"
-	ForbiddenReason         = "SysctlForbidden"
+	// ForbiddenReason represents reason to forbidden sysctl
+	ForbiddenReason = "SysctlForbidden"
 )
 
-// patternWhitelist takes a list of sysctls or sysctl patterns (ending in *) and
+// PatternWhitelist takes a list of sysctls or sysctl patterns (ending in *) and
 // checks validity via a sysctl and prefix map, rejecting those which are not known
 // to be namespaced.
-type patternWhitelist struct {
+type PatternWhitelist struct {
```

Comment:
Why is this being exported?

Reply:
how about keeping this unexported and just take other changes? but that we have to keep pkg/kubelet/sysctl in the hack/.golint_failures.

Reply:
The better fix then is to change NewWhitelist to return a `lifecycle.PodAdmitHandler`